tests/setup.sh

```diff
@@ -1,15 +1,51 @@
-#!/bin/bash -e
-# Copyright (C) 2024 Simo Sorce <simo@redhat.com>
+#!/bin/bash -ex
+# Copyright (C) 2024 Simo Sorcen <simo@redhat.com>
 # SPDX-License-Identifier: Apache-2.0
 
 source "${TESTSSRCDIR}/helpers.sh"
 
-if ! command -v p11tool &> /dev/null
-then
-    echo "GnuTLS utils are required"
-    exit 0
+if [ $# -ne 1 ]; then
+    echo "Usage setup.sh <tokentype>"
+fi
+
+TOKENTYPE=$1
+
+# Temporary dir
+TMPPDIR="${TESTBLDDIR}/${TOKENTYPE}"
+if [ -d "${TMPPDIR}" ]; then
+    rm -fr "${TMPPDIR}"
+fi
+mkdir "${TMPPDIR}"
+
+# Token data dir
+TOKDIR="$TMPPDIR/tokens"
+if [ -d "${TOKDIR}" ]; then
+    rm -fr "${TOKDIR}"
+fi
```

Comment:
Is this check needed when the tmpdir is removed and freshly created few lines above anyway?

Reply:
ha no, I just copied the block without questioning it :)